demo/micropython-run.ts

```diff
@@ -12,20 +12,30 @@ const args = minimist(process.argv.slice(2), {
     'image', // UF2 image to load; defaults to "rp2-pico-20210902-v1.17.uf2"
     'expect-text', // Text to expect on the serial console, process will exit with code 0 if found
   ],
-  boolean: 'gdb', // start GDB server on 3333
+  boolean: [
+    'gdb', // start GDB server on 3333
+    'circpy', // use CircuitPython instead of MicroPython
```

Comment:
why not use the full name, `circuitpython`?

Reply:
No reason, I tend to default to shorter flag names but I can change it - seems more sensible in full!

Reply:
Yes please